src/libpriv/rpmostree-core.c

```diff
@@ -1269,6 +1270,36 @@ rpmostree_context_prepare_install (RpmOstreeContext    *self,
       hy_goal_install (goal, pkg);
     }
 
+  { GPtrArray *repos = dnf_context_get_repos (hifctx);
```

Comment:
Could we do this in `download_metadata` instead? Just so we print it as soon as the information is available.

This will also be triggered in the compose path, right? Not a big deal, though I don't think we actually write anything to the journal right now when composing (and we already print that information on stdout there). Maybe make `download_metadata()` take a `to_stdout` to toggle between this and what treecompose is currently printing right now?